integration-tests/resteasy-reactive-kotlin/src/main/kotlin/io/quarkus/it/resteasy/reactive/kotlin/ReactiveGreetingResource.kt

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.it.resteasy.reactive.kotlin
+
+import kotlinx.coroutines.delay
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+import javax.ws.rs.Produces
+import javax.ws.rs.core.MediaType
+
+@Path("/hello-resteasy-reactive")
+class ReactiveGreetingResource {
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    suspend fun hello(): String {
+        delay(50)
+        return "Hello RestEASY Reactive"
+    }
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("/standard")
+    fun standard(): String {
+        return "Hello RestEASY Reactive"
+    }
+
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    @Path("/{name}")
+    suspend fun hello(name:String): String {
+        delay(50)
```

Comment:
I think it would be much better if this method called other suspending methods (for example methods doing some network IO)

Reply:
I really want a proper test here as I explained above

Reply:
oh, right.  i missed this comment in the noise.  What would you like to see exactly?  `delay` is a `suspend` function which can only be called from a suspended context.  It won't even compile otherwise.  I did have the delay up to 5000 ms which worked but made the tests take forever.  This call, as it exists now, at least shows that other `suspend` functions can be successfully called without overly delaying the test run.

Reply:
Sorry, I deleted your comment by accident :(

Reply:
Sure, but as this is the only test we have, I would much have it be more realistic - make an http call or two for example and await the result

Reply:
ha! i thought maybe i'd offended you somehow.  at any rate, how does loading up red hat's home page sound for an IO test?  it is, itself not a suspended call (using `URL`) but is at least some network IO.

Reply:
I was trying to erase a comment of mine (which was in pending  and I wanted to make a single comment and I mistakenly deleted yours :().
I was thinking of hitting maven central, as it will almost certainly work if the test has reached that part, whereas Red Hat's home page could theoretically be down.

In any case, any network IO will do :)

Reply:
And maybe you can even leave in the delay for good measure :)

Reply:
added!

Reply:
Thanks!